src/global.spec.ts

```diff
@@ -1,26 +1,36 @@
-import client from "./shared/api/cbioportalClientInstance";
-import Sinon from "sinon";
 import CBioPortalAPI from "./shared/api/generated/CBioPortalAPI";
+import request from 'superagent';
 
 let clientStubs:any[];
-before(()=>{
-    clientStubs = stubClient();
-});
 
-after(()=>{
-    for (const stub of clientStubs) {
-        stub.restore();
-    }
-});
+clientStubs = stubClient();
+
+request.get = (function(){
```

Comment:
We can't use sinon to stub these because then sinon won't us stub them again if necessary for individual tests. 
also, uncessary i think to restore here because we don't ever want real implementation to run